Production/python/sampleConfig.py

```diff
@@ -41,6 +45,10 @@
                  'Emb_18D' : '102X_dataRun2_Prompt_v15',
                  'MC_Phase2_110X' : '110X_mcRun4_realistic_v3',
                  'MC_Phase2_111X' : 'auto:phase2_realistic_T15',
+                 'UL18': '106X_upgrade2018_realistic_v15_L1v1',
+                 'UL17': '106X_mc2017_realistic_v8',
+                 'UL16': '106X_mcRun2_asymptotic_v17',
+                 'UL16APV': '106X_mcRun2_asymptotic_preVFP_v11'
```

Comment:
- there are already `MC_UL18` and `RunUL18` in the list. Please, remove duplicates and follow the naming notation
- data and MC have different GTs
- use GTs recommended in this twiki https://twiki.cern.ch/twiki/bin/view/CMS/PdmVRun2LegacyAnalysis